Governance/publish/publish-options.md

```diff
@@ -1,27 +1,27 @@
 # Publishing options
-Once the data is created and ready for publication, **where** can it be published? 
+Before starting preparing the data files for publication, it is important to individuate **where** it can be published.
 
 One of the main factors is what services are available, which is largely determined by the researcher's institution and/or organisation.
 Another important consideration is the kind of data, depending on format, size etc. 
 In some cases, depending on the dataset, using a public repository is suitable and often the easiest option.
 On the other end of the scale, some data could be produced on purpose or simply be suitable to contribute to a large-scale project that has its own publishing procedure established.
 The most common use cases for our community are covered here.
 
-::::{tab-set}
-:::{tab-item} BoM 
+````{tab-set}
+```{tab-item} BoM 
 Insert here pathways for BoM researchers...
 
 
 
-:::
+```
 
-:::{tab-item} CSIRO 
+```{tab-item} CSIRO 
 Data, software and links to external data holdings (such as NCI) can be added to the [CSIRO Data Access Portal](https://data.csiro.au/) (DAP) by **CSIRO staff** only. More information on creating metadata records and uploading files to the CSIRO DAP can be found on the [DAP Help Guide](https://confluence.csiro.au/display/dap/Deposit+and+Manage+Data) (staff only access).

```

Comment:
I think we can leave them both we need the first anyway to see what DAP is and it actually help also authors understanding how the data is accessed so it can't hurt. I've updated the links.